### PR TITLE
Add other possible name as a demo xcodeproj

### PR DESCRIPTION
**Pull request by pixyzehn**
For instance,
https://github.com/Alamofire/Alamofire/tree/master/Example

### CODE REVIEW

File: Sources/TestDrive.swift [hunk 1]
```diff
@@ -221,7 +221,12 @@ class PackageLoader {
         try shellOut(to: "git submodule update --init --recursive --quiet", at: repositoryFolder.path)
 
         for subfolder in repositoryFolder.makeSubfolderSequence(recursive: true) {
-            if subfolder.extension == "xcodeproj" && !subfolder.name.lowercased().contains("demo") {
+            let lowercasedName = subfolder.name.lowercased()
+            if subfolder.extension == "xcodeproj"
+                && !lowercasedName.contains("demo")
+                && !lowercasedName.contains("sample")
+                && !lowercasedName.contains("example") {
```
Comment (JohnSundell) [inline Sources/TestDrive.swift]: How about breaking this out into a method/function instead of having such a complex if-statement? 🙂

Comment (pixyzehn) [inline Sources/TestDrive.swift]: That would be great👍 I'll implement a method later:)

Comment (pixyzehn) [inline Sources/TestDrive.swift]: @JohnSundell I added a new method:)

File: Sources/TestDrive.swift [hunk 2]
```diff
@@ -253,6 +248,16 @@ class PackageLoader {
...
+
+    private func isValidatedFolder(named name: String) -> Bool {
+        let lowercasedName = name.lowercased()
+        for invalidName in ["demo", "sample", "example"] {
+            if lowercasedName.contains(invalidName) {
+                return false
+            }
+        }
+        return true
+    }
```
Comment (NanDotWang) [inline Sources/TestDrive.swift]: Maybe we can consider something simpler for this function?
```swift
let containsInvalidNames = ["demo", "sample", "example"].map { name.lowercased().contains($0) } .contains(true)

return !containsInvalidNames
```

Comment (pixyzehn) [inline Sources/TestDrive.swift]: @NanDotWang That's right, but I'd like to return earlier if an `invalidName` is contained.
In this case, if lowercasedName is `Demo`, you can return earlier. It means you don't need to loop all cases.

Comment (NanDotWang) [inline Sources/TestDrive.swift]: @pixyzehn, really good point :D

Thank you for giving me feedback :)

**Comment by pixyzehn** [comment]
You're welcome!

**Pull request merged**